core/src/main/java/io/questdb/ServerMain.java

```diff
@@ -25,701 +25,187 @@
 package io.questdb;
 
 import io.questdb.cairo.*;
-import io.questdb.cutlass.http.HttpServer;
-import io.questdb.cutlass.json.JsonException;
-import io.questdb.cutlass.line.tcp.LineTcpReceiver;
-import io.questdb.cutlass.line.udp.LineUdpReceiver;
-import io.questdb.cutlass.line.udp.LinuxMMLineUdpReceiver;
-import io.questdb.cutlass.pgwire.PGWireServer;
+import io.questdb.cutlass.Services;
 import io.questdb.cutlass.text.TextImportJob;
 import io.questdb.cutlass.text.TextImportRequestJob;
 import io.questdb.griffin.DatabaseSnapshotAgent;
 import io.questdb.griffin.FunctionFactory;
 import io.questdb.griffin.FunctionFactoryCache;
+import io.questdb.griffin.SqlException;
 import io.questdb.griffin.engine.groupby.vect.GroupByJob;
 import io.questdb.griffin.engine.table.LatestByAllIndexedJob;
-import io.questdb.jit.JitUtil;
 import io.questdb.log.Log;
 import io.questdb.log.LogFactory;
-import io.questdb.log.LogRecord;
 import io.questdb.mp.WorkerPool;
-import io.questdb.network.NetworkError;
+import io.questdb.mp.WorkerPoolManager;
 import io.questdb.std.*;
-import io.questdb.std.datetime.millitime.Dates;
-import io.questdb.std.str.NativeLPSZ;
-import io.questdb.std.str.Path;
-import sun.misc.Signal;
 
-import java.io.*;
-import java.net.*;
-import java.util.*;
-import java.util.concurrent.atomic.AtomicInteger;
-import java.util.zip.ZipEntry;
-import java.util.zip.ZipInputStream;
+import java.util.ServiceLoader;
+import java.util.concurrent.atomic.AtomicBoolean;
 
-public class ServerMain {
-    private static final String VERSION_TXT = "version.txt";
-    private static final String PUBLIC_ZIP = "/io/questdb/site/public.zip";
+public class ServerMain implements QuietCloseable {
```

Comment:
I have renamed it to Server.

I would prefer to leave main in Server, rather than creating a Main.java that contains it.

Reply:
Well, to avoid changes in Docker entrypoint and other places, we could have both `Server` and `ServerMain`. The latter would contain only the main method.